[3384][ADD] stock_picking_product_type (#21)
qrtl/axls-custom: stock_move_product_type/readme/DESCRIPTION.rst

> This module displays product type in lines of stock move.

Reviewer: ```suggestion
This module displays product type in stock moves in the picking form.
```
